examples/data-generators/ecommerce/ecommerce-policy.json

```diff
@@ -2,6 +2,11 @@
   "name": "ecommerce-policy",
   "duration": 10000,
   "saveRawData": "false",
+  "checkpointDir": "checkpoint",
+  "timeBucket": "minute",
+  "checkpointGranularity": "minute",
```

Comment:
This value is milliseconds. Is the additional time that you want extend the updateStateByKey.